src/Symfony/Component/AssetMapper/ImportMap/ImportMapGenerator.php

```diff
@@ -0,0 +1,245 @@
+<?php
+
+namespace Symfony\Component\AssetMapper\ImportMap;
+
+use Symfony\Component\AssetMapper\AssetMapperInterface;
+use Symfony\Component\AssetMapper\Exception\LogicException;
+use Symfony\Component\AssetMapper\MappedAsset;
+use Symfony\Component\AssetMapper\Path\PublicAssetsPathResolverInterface;
+
+/**
+ * Provides data needed to write the importmap & preloads.
+ */
+class ImportMapGenerator
```

Comment:
Class is a copy-and-paste of methods from `ImportMapManager` with very few changes